config/php-fpm/docker-php-ext-xdebug.ini

```diff
@@ -1 +1,6 @@
 zend_extension=/usr/local/lib/php/extensions/no-debug-non-zts-20151012/xdebug.so
+
+[Xdebug]
+xdebug.remote_enable = 1
+xdebug.remote_log = /var/log/xdebug/xdebug.log
+xdebug.remote_host = 172.18.0.1
```

Comment:
Check the updated readme. On Windows that IP is your network bridge adapter IP. It should be in the same range as your machine, so `192.168.x.x`